src/EOFTestsFiller/EIP4200/validInvalidFiller.yml

```diff
@@ -60,7 +60,7 @@ validInvalid:
          EOF1I4200_0042 (Invalid) EOF code containing RJUMPV with target CALLF immediate
 
   forks:
-    - ">=Shanghai"
+    - ">=Cancun"
```

Comment:
EOF tests are not moved to Prague yet

Reply:
This ws correct expectation, it's valid in Prague, but invalid in Cancun and before